source/Halibut.Tests/Queue/Redis/MessageSerialiserAndDataStreamStorageExceptionObserverTest.cs

```diff
@@ -0,0 +1,131 @@
+#if NET8_0_OR_GREATER
```

Comment:
It's been a while since I've been in this code base, but why are we doing this for .NET 8 or higher only?

Reply:
Good question, because the queue is only supported in net8 or higher. The Redis PRQ does not exist for net48. Because screw net48!